CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v9.4.1
+
+### Bug Fixes
+
+- Update the logic to get path of access tokens through AZURE_ACCESS_TOKEN_FILE.
```

Comment:
Mind rephrasing this to make clear that the previous default still works?

Reply:
Yes.